profiling/src/io.rs

```diff
@@ -424,20 +444,22 @@ unsafe extern "C" fn observed_fread(
     buf: *mut c_void,
     size: usize,
     n: usize,
-    fp: *mut libc::FILE,
+    fd: *mut libc::FILE,
 ) -> usize {
     let start = Instant::now();
-    let len = ORIG_FREAD(buf, size, n, fp);
+    let len = ORIG_FREAD(buf, size, n, fd);
```

Comment:
I recommend reverting that, reading fp is better than fd for something that isn't actually a fd

Reply:
I think going with `stream` for `fread()`/`fwrite()` would match with[ Rust's `libc::fread()` and `libc::fwrite()` definition](https://docs.rs/libc/latest/libc/fn.fread.html). Okay with you?

Reply:
Also fine, yes.